### PR TITLE
Debug commands

### PR DESCRIPTION
**Pull request by giggsoff**
Initial support for eden debug:
```
./eden utils debug start
./eden utils debug stop
./eden utils debug save flamegraph.svg
./eden utils gitupload flamegraph.svg <git repo in notation https://GIT_LOGIN:GIT_TOKEN@GIT_REPO> <branch> [directory on git]
```

### CODE REVIEW

**Comment by mydatascience** [comment]
Suggesting eden utils debug upload  -> eden utils gitupload

File: cmd/debug.go [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+			eveRemote = viper.GetBool("eve.remote")
+			eveRemoteAddr = viper.GetString("eve.remote-addr")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		eveIP := getEVEIP()
+		if eveIP == "" {
+			log.Fatal("Np EVE IP")
+		}
```
Comment (mydatascience) [inline cmd/debug.go:85]: May we have it inside getEVEIP?

File: cmd/debug.go [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+		if eveRemote || eveRemoteAddr == "" {
+			if !cmd.Flags().Changed("eve-ssh-port") {
+				eveSSHPort = 22
+			}
+		}
+		if _, err := os.Stat(eveSSHKey); !os.IsNotExist(err) {
```
Comment (mydatascience) [inline cmd/debug.go:91]: Please DoesNotExist...

File: cmd/debug.go [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+		if viperLoaded {
+			eveSSHKey = utils.ResolveAbsPath(viper.GetString("eden.ssh-key"))
+			extension := filepath.Ext(eveSSHKey)
+			eveSSHKey = strings.TrimRight(eveSSHKey, extension)
+			eveRemote = viper.GetBool("eve.remote")
+			eveRemoteAddr = viper.GetString("eve.remote-addr")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
```
Comment (mydatascience) [inline cmd/debug.go:81]: This thing does the same as eden eve ssh AFAIU 
Maybe reuse eden eve ssh functions? Or use this there?

File: cmd/debug.go (hunk 2 above)
Comment (giggsoff) [reply cmd/debug.go:91]: Sorry, it is library function.

File: cmd/debug.go (hunk 3 above)
Comment (giggsoff) [reply cmd/debug.go:86]: Actually, in debug.go we run commands in foreground, in background, scp/ssh, docker run... I do not thing that we can reuse eve ssh efficiently.

File: cmd/debug.go (hunk 2 above)
Comment (mydatascience) [reply cmd/debug.go:91]: ok

**Pull request merged**